curriculum/challenges/english/blocks/workshop-fortune-teller-app/68c38c3275a149b3cfed312a.md

```diff
@@ -0,0 +1,400 @@
+---
+id: 68c38c3275a149b3cfed312a
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+In this Workshop, you are going to build a fortune-telling-app step by step.
+
+Begin by declaring an interface named Card. Inside it, add the following properties    `name` and `name_short` with type string and for `value` you should have type union of `string | number`
```

Comment:
This feedback is for missing backticks for the `Card` interface and rewording the instructions a little bit. 

```suggestion
Begin by declaring an interface named `Card`. Inside the `Card` interface, add the following properties: `name`, `name_short` and `value`. The `name`, and `name_short` properties should have the types set to `string` and the `value` should have a union type set to `string | number`.
```